core/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -379,6 +383,26 @@ public boolean trackScores() {
         return trackScores;
     }
 
+
+    /**
+     * The sort values that indicates which docs this request should "search after".
+     * The sort values of the search_after must be equal to the number of sort fields in the query and they should be
+     * of the same type (or parsable as such).
+     * Defaults to <tt>null</tt>.
+     */
+    public SearchAfterBuilder searchAfter() {
+        return searchAfterBuilder;
+    }
+
+    /**
+     * Adds a search_after builder.
+     *
+     */
+    public SearchSourceBuilder searchAfter(SearchAfterBuilder searchAfterBuilder) {
+        this.searchAfterBuilder = searchAfterBuilder;
+        return this;
+    }
```

Comment:
I'm wondering that maybe we should not expose SearchAfterBuilder at all to the client API and make it internal to SearchSourceBuilder? So SearchSourceBuilder would still hold a SearchAfterBuilder instance, but the two above methods would look like:

```java
    public Object[] searchAfter() {
        return searchAfterBuilder.getSortValues();
    }
    public SearchSourceBuilder searchAfter(Object[] values) {
        this.searchAfterBuilder = new SearchAfterBuilder(values);
        return this;
    }
```